tools/resources/resources.json

```diff
@@ -128,7 +128,12 @@
         "webapi/webapi-vehicleinfo-ivi-tests/resources",
         "webapi/webapi-webcl-nonw3c-tests/resources",
         "webapi/webapi-webrtc-w3c-tests/resources",
-        "webapi/webapi-webspeech-w3c-tests/resources"
+        "webapi/webapi-webspeech-w3c-tests/resources",
```

Comment:
@zqzhang There is too many sub-app, and some apps have packed the resources in app. If we maintain all sub app resource in copy.py, resource.json will be very large. I want to keep the resource.json only maintain test suites resources, can we copy them by manual to sub app if we need debug them?